.environment

```diff
@@ -0,0 +1,6 @@
+# Statements in this file will be executed (sourced) by the shell in SSH
+# sessions, in deploy hooks, in cron jobs, and in the application's runtime
+# environment.
+
+# Allow executable app dependencies from Composer to be run from the path.
+export PATH=/app/vendor/bin:$PATH
```

Comment:
`/app/vendor/bin`? Anything in this file seems like it should just reference the project path. All systems, for instance, won't have `/app`. Could you add this to your `~/.bashrc`, `.zshrc` or similar?

Another solution around this could be to use [Composer Scripts](https://getcomposer.org/doc/articles/scripts.md), since they [automatically add vendor/bin to PATH](https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands).

```
{
    "scripts": {
        "test": "phpunit"
    }
}
```
```
composer test
```